src/main/java/org/radarcns/management/service/ProjectService.java

```diff
@@ -65,7 +64,7 @@ public ProjectDTO save(ProjectDTO projectDto) {
     public Page findAll(Boolean fetchMinimal, Pageable pageable) {
         Page<Project> projects = projectRepository.findAllWithEagerRelationships(pageable);
         if (!fetchMinimal) {
-            return projects.map(projectMapper::projectToProjectDTO);
+            return projects.map(projectMapper::projectToProjectDTOReduced);
```

Comment:
The projectToMinimalProjectDetailsDTO also has part of the project data. Maybe we can remove the fetchMinimal condition and just use a single DTO?

Reply:
Minimal project data only includes name and ID, but not the information needed to print the projects e.g. on the home page.